ui/src/test/java/com/hyperwallet/android/ui/transfermethod/TransferMethodUtilsTest.java

```diff
@@ -134,4 +140,48 @@ public void testGetStringFontIcon_returnsDefaultValue() {
         verify(mResources, times(2)).getIdentifier(anyString(), anyString(), anyString());
         verify(mContext, times(1)).getString(anyInt());
     }
+
+    @Test
+    public void testGetSecondLine_returnsPayPalSecondLine() throws JSONException {
```

Comment:
For all those test cases, just use the builder classes to create instances of PayPal, BankAccount etc... easier than loading the json file. Just provide the data in the field which will be used to retrieve the value from. I.e 
`new HyperwalletBankAccount.Builder().bankAccountId("8017110254").build()`

Reply:
Thanks, now it much more handy with a builder.
refactored them.